### PR TITLE
Wdi10 update

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,41 @@
-# api-intro-with-ajax
+<!-- TO DO -->
+<!-- Examples of APIs they've used so far? -->
```
Comment (jshawl) [inline README.md]: DOM, jQuery are both apis

File: README.md [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
+<!-- Are there popular XML-only APIs? -->
```
Comment (jshawl) [inline README.md]: twilio?

File: README.md [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+<!-- Provide additional info on what exactly a promise is. -->
+<!-- Show them what exactly a promise looks like. -->
+<!-- Give a beginner's definition of HTTP methods and CRUD -->
```
Comment (jshawl) [inline README.md]: went over GET and POST in php class, so those words are familiar

File: README.md [hunk 4]
```diff
@@ -117,32 +126,51 @@ While the majority of APIs are free to use, many of them require an API "key" th
 * With key: [http://api.giphy.com/v1/gifs/search?q=funny+cat&api_key=dc6zaTOxFJmzC](http://api.giphy.com/v1/gifs/search?q=funny+cat&api_key=dc6zaTOxFJmzC)
 
 **It is very important that you not push your API keys to a public Github repo.**
-> Check out [Figaro](https://github.com/laserlemon/figaro) or research some additional ways to hide your keys
 
-## AJAX (5 mins)
+> This is especially true when working with [Amazon Web Services (AWS)](https://aws.amazon.com/). Here's an example of a [stolen key horror story](https://wptavern.com/ryan-hellyers-aws-nightmare-leaked-access-keys-result-in-a-6000-bill-overnight).
```
Comment (jshawl) [inline README.md:122]: this happened to Frish!!! form last cohort!

File: README.md [hunk 5]
```diff
@@ -117,32 +126,51 @@ While the majority of APIs are free to use, many of them require an API "key" th
...
+**AJAX**, which stands for "Asynchronous Javascript and XML," is the method through which we are able to make HTTP requests. The standard requests we will be making are `GET` `POST` `PUT` `PATCH` and `DELETE`.
 
-**So we know what an API is, now what?**
+| Type of Request | What's It Do? | An Example URL |
+|-----------------|---------------|----------------|
+| `GET`  | Read | http://www.yearbook.com/students |
+| `POST` | Create | http://www.yearbook.com/students |
+| `PUT` | Update | http://www.yearbook.com/students/4 |
+| `PATCH` | Update | http://www.yearbook.com/students/4 |
+| `DELETE` | Delete | http://www.yearbook.com/students/4 |
```
Comment (jshawl) [inline README.md]: maybe use https://swapi.co/ for real examples they can play with

Comment (jshawl) [inline README.md]: is GET only, but touches on id in url

File: README.md [hunk 6]
```diff
@@ -152,287 +180,242 @@ $.ajax({
...
+</details>
 
 ### Promises
 
-But, that is not all we need to complete the request. The AJAX method also returns what we call promises. In short, these tell the `$.ajax` request what to do if the request is successful, fails, or both. In this case, we are going to add 3:
+Now we need to tell our AJAX method what to do next. We can do this by doing something with the its return value, which comes in the form of a **promise**. We can use **promise methods** to tell the `$.ajax` request what to do if the request is successful or not. In this case, we are going to add three...
+
+#### `.done`
+
+A promise method for when the AJAX call is successful...
```
Comment (jshawl) [inline README.md:213]: maybe compare promise to callback syntax here

File: README.md [hunk 7]
```diff
@@ -152,287 +180,242 @@ $.ajax({
...
-    data: {artist: {photo_url: "www.google.com", name: "bob", nationality: "bob"}},
+    data: {
+      artist: {
+        name: "Limp Bizkit",
+        photo_url: "http://sessionsx-magazine.itibitiventuresi.netdna-cdn.com/wp-content/uploads/2016/01/Featured-What-Happened-To-Limp-Bizkit-.jpg",
+        nationality: "USA"
+      }
+    },
     dataType: 'json',
     url: "http://localhost:3000/artists"
```
Comment (jshawl) [inline README.md]: should be heroku, right?

File: README.md [hunk 8]
```diff
@@ -152,287 +180,242 @@ $.ajax({
...
-
-Here's an example of a successful `200 OK` API call...
-
-![Postman screenshot success](http://i.imgur.com/2TADr4J.png)
-
-And here's an example of an unsuccessful `403 Forbidden` API call. Why did it fail?
-
-![Postman screenshot fail](http://i.imgur.com/r3nIhGH.png)
-
-## Resources:
```
Comment (jshawl) [inline README.md:396]: what is people are hungry for more?

File: README.md (hunk 1 above)
Comment (amaseda) [inline README.md]: :+1:

File: README.md (hunk 4 above)
Comment (amaseda) [inline README.md:122]: :+1:

File: README.md (hunk 3 above)
Comment (amaseda) [inline README.md]: :+1:

File: README.md (hunk 5 above)
Comment (amaseda) [inline README.md]: :+1:

File: README.md (hunk 6 above)
Comment (amaseda) [inline README.md:213]: :+1:

File: README.md (hunk 7 above)
Comment (amaseda) [inline README.md]: :+1:

File: README.md (hunk 8 above)
Comment (amaseda) [inline README.md:396]: :+1:

**Pull request merged**